src/librawspeed/decoders/NefDecoder.cpp

```diff
@@ -76,7 +76,7 @@ RawImage NefDecoder::decodeRawInternal() {
   TiffEntry *offsets = raw->getEntry(STRIPOFFSETS);
   TiffEntry *counts = raw->getEntry(STRIPBYTECOUNTS);
 
-  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100 ") { /**Sigh**/
+  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100") { /**Sigh**/
```

Comment:
no, looking at the code, the sigh was about the d100 in general.
nikon is well-known for their, uhm, mindless raw format